db/fixtures/ae_datastore/ManageIQ/Container/Service/Provisioning/StateMachines/Provision.class/__methods__/check_provisioned.rb

```diff
@@ -0,0 +1,37 @@
+#
+# Description: This method checks to see if OpenShift container has been provisioned
+# and refresh the provider
+#
+
+class OpenshiftCheckProvisioned
```

Comment:
@bzwei 
Based on some of the feedback we got from Jason we have decided to add namespace to Automate Methods.
We have a sample here
https://github.com/ManageIQ/manageiq/blob/master/db/fixtures/ae_datastore/ManageIQ/Cloud/Orchestration/Operations/Methods.class/__methods__/available_availability_zones.rb